.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -31,7 +31,7 @@ Note: remove the backticks (``) from the link.
 -->
 
 - [x] Release `[<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)` is cut as a new canary release
-- [ ] Release <RELEASE_NUMBER> pushed to dev channel (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> pushed to experimental channel (<CL submit time>)
 - [ ] Release <RELEASE_NUMBER> pushed to 1% (<CL submit time>)
 - [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
```

Comment:
You'd eventually have to replace "production" with "stable" (can be in another PR, or roped into this one if it's straightforward).

Reply:
Done.